src/status_im2/contexts/chat/messages/view.cljs

```diff
@@ -27,7 +29,7 @@
     ;; and will call system back button action
     true))
 
-(defn page-nav
+(defn old-page-nav
```

Comment:
why do we need an old one?

Reply:
Removed it since it's not needed anymore